key-management-scenarios.md

```diff
@@ -0,0 +1,19 @@
+# Notary Key Management Scenarios
+In order to allow for end to end signing of artifacts, Notary v2 will require the use of cryptographic keys. These scenarios define how these keys will be generated, distributed, and trusted in Notary v2.
+
+## Scenarios
+
+### Scenario #1: A developer signs a package for the first time
+The first time a developer signs a package, they will need to set up a signing key and their local environment.
+  1. Generate a developer signing key.
+  1. Create a local repository that delegates to the developer key notary-tool --create-local-repository, notary-tool --delegate developer-key.pub
+
+### Scenario #2: A developer is given permission to upload to a registry (for the first time)
+The first time a developer uploads to a registry, they need to establish that they are a trusted developer.
+  1. Add the developer key to delegating metadata on the registry. The delegating metadata may be top-level targets metadata, an organization-hosted targets metadata file (see TAP 13 for how organization-specific targets metadata can be used), or a different targets metadata file.
```

Comment:
Line 12 - great.
Again, way too detailed for implementation. 
Can we please avoid specific assumptions about design in the scenarios, such as references to TAP x